techempower/kld_manifests/postgres.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
```

Comment:
Can you update the read me with manifests and kld_manifests

Reply:
Done.